ja/orm/entities.rst

```diff
@@ -318,14 +318,14 @@ mass-assignment が保護されたフィールドへのアクセスすること
 -----------------------------------
 
 データベスに既に存在する行をエンティティーが表しているのか知る必要がある時があります。
-``isNew()`` でそれが分かります。::
+``isNew()`` でそれが分かります。:
```

Comment:
Its only not necessary if there is a `::` on a line below. For example,

```rst
Some words::

    $this->is->code();

More words

and 

```rst
Some words

::

    $this->is->code();

More words
```

Will both correctly generate a formatted code block. But what you have right now will not. If you want to test the build output locally. The README has instructions on how to install the dependencies and run sphinx.